extras/mmu.py

```diff
@@ -2006,26 +2006,52 @@ def _get_print_status(self):
             self._log_trace("Determined print status as: %s from %s" % (print_status, source))
             return print_status
 
+    def _wait_for_target_temp(self):
+        target_temp = self.printer.lookup_object(self.extruder_name).heater.target_temp
+        current_temp = self.printer.lookup_object(self.extruder_name).get_status(0)['temperature']
+
+        if abs(target_temp - current_temp) < 1:
+            self._log_debug("Waiting for extruder to reach target temperature (%.1f)" % target_temp)
+            self.gcode.run_script_from_command("TEMPERATURE_WAIT SENSOR=extruder MINIMUM=%.1f MAXIMUM=%.1f" % (target_temp - 1, target_temp + 1))
+
     # Ensure we are above desired or min temperature and that target is set
-    def _set_above_min_temp(self, target_temp=-1):
+    def _set_above_min_temp(self, target_temp_override=-1):
```

Comment:
I think I'm going to rename this to better capture the new logic. It's no longer about the min temp, it's about waiting for a safe temp